src/cleanup_rules/swift/rules.toml

```diff
@@ -383,6 +383,40 @@ replace_node = "if_else_block"
 replace = "@alternatives"
 is_seed_rule = false
 
+
+# Before:
+# if true, anything { }
+# if true, someOptional { }
```

Comment:
Let's add the construct matched by the third clause in the query

Reply:
Here in this rule there are three choices -`@ifstl` . We have documented two patterns here in the comment, can we document the missing one ?